src/framework/preferences/js/Panels.js

```diff
@@ -788,12 +847,99 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
+                    },
+                    range: "{fluid.prefs.panel.lineSpace}.options.range",
+                    buttonOptions: "{fluid.prefs.panel.lineSpace}.options.buttonOptions"
+                }
+            }
+        },
+        buttonOptions: {
+            stepMultiplier: 10
+        }
+    });
+
+    /*********************************
+     * Preferences Editor Blue Color Filter *
+     *********************************/
+
+    /**
+     * A sub-component of fluid.prefs that renders the "blue color filter" panel of the user preferences interface.
+     */
+    fluid.defaults("fluid.prefs.panel.blueColorFilter", {
+        gradeNames: ["fluid.prefs.panel", "fluid.contextAware"],
+        contextAwareness: {
+            responsiveAware: {
+                checks: {
+                    responsive: {
+                        contextValue: "{fluid.responsiveCheck}",
+                        gradeNames: "fluid.prefs.blueColorFilter.textfieldButtons"
+                    }
+                },
+                defaultGradeNames: "fluid.prefs.blueColorFilter.textfieldSlider"
+            }
+        },
+        preferenceMap: {
+            "fluid.prefs.blueColorFilter": {
+                "model.blueColorFilter": "default",
+                "range.min": "minimum",
+                "range.max": "maximum"
+            }
+        },
+        // The default model values represent both the expected format as well as the setting to be applied in the absence of values passed down to the component.
+        // i.e. from the settings store, or specific defaults derived from schema.
+        // Note: Except for being passed down to its subcomponent, these default values are not contributed and shared out
+        range: {
+            min: 0,
+            max: 1
+        },
+        selectors: {
+            blueColorFilter: ".flc-prefsEditor-blue-color-filter",
+            label: ".flc-prefsEditor-blue-color-filter-label",
+            inactiveMessage: ".flc-prefsEditor-blue-color-filter-inactive-message"
+        },
+        selectorsToIgnore: ["blueColorFilter"],
         protoTree: {
-            label: {messagekey: "lineSpaceLabel"},
-            narrowIcon: {messagekey: "lineSpaceNarrowIcon"},
-            wideIcon: {messagekey: "lineSpaceWideIcon"},
+            label: {messagekey: "blueColorFilterLabel"},
             multiplier: {messagekey: "multiplier"},
-            lineSpaceDescr: {messagekey: "lineSpaceDescr"}
+            blueColorFilterDescr: {messagekey: "blueColorFilterDescr"},
+            inactiveMessage: {messagekey: "blueColorFilterInactiveMessage"}
+        }
+    });
+
+    fluid.defaults("fluid.prefs.blueColorFilter.textfieldSlider", {
+        selectors: {
+            multiplier: ".flc-prefsEditor-multiplier",
+            blueColorFilterDescr: ".flc-prefsEditor-blue-color-filter-descr"
+        },
+        components: {
+            textfieldSlider: {
```

Comment:
Once the common grade component created for text size and line space components, it should also be reused here.